src/plugins/discover/public/application/components/default_discover_table/_table_header.scss

```diff
@@ -4,7 +4,7 @@
 
   // nested for specificity
   .docTableHeaderField {
-    padding: calc($euiSizeM / 2); // corresponds to DataGrid medium cellPadding
+    padding: $euiSizeS;
```

Comment:
callout: this was intentionally lowered for density. not going to block on this because i'm not sure why we're making this change, but preference was still for this to be denser, similar to competitor data tables spacing

also, haven't gone through this entire cr yet, but double check the hover behavior on the cells as there was some change to that after adjusting density here

Reply:
Checked with UX and the change is good to stay.